flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/table/ReducingUpsertWriterTest.java

```diff
@@ -51,6 +51,7 @@
 import static org.apache.flink.types.RowKind.INSERT;
 import static org.apache.flink.types.RowKind.UPDATE_AFTER;
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertNull;
```

Comment:
Per the [Flink code quality guidelines](https://flink.apache.org/contributing/code-style-and-quality-common.html) the usage of JUnit assertions should not be used